.github/workflows/generate-release.yml

```
@@ -1,11 +1,276 @@
+      llamastack_version: ${{ steps.validate.outputs.llamastack_version }}
+      release_branch: ${{ steps.validate.outputs.release_branch }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          fetch-depth: 0  # Need full history to check existing tags
+
+      - name: Validate inputs
+        id: validate
+        shell: bash
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:27:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)